src/screens/Friend/GroupInfoScreen.js

```diff
@@ -71,6 +82,7 @@ const GroupInfoScreen = ({ navigation, route }) => {
         isExpanded: isPendingMemberExpanded,
         changeExpanded: setIsPendingMemberExpanded
```

Comment:
I don't know why but expanding "pending member" will cause the "members" expansion animation to retrigger